bin/startPipelineV2.sh

```diff
@@ -0,0 +1,409 @@
+####################################### 
+### Turned the track N trace for now
+##
+		## creating jobs entity
+                #echo -e "job\tproject_job\tproject\tstarted_date\tfinished_date\tstatus" >  ${LOGDIR}/${_project}/jobsPerProject.tsv
+                #grep 'processJob' submit.sh | tr '"' ' ' | awk -v pro=$PROJECT '{OFS="\t"} {print $2,pro"_"$2,pro,"","",""}' >>  ${LOGDIR}/${_project}/jobsPerProject.tsv
+
+                #CURLRESPONSE=$(curl -H "Content-Type: application/json" -X POST -d "{"username"="${USERNAME}", "password"="${PASSWORD}"}" https://${MOLGENISSERVER}/api/v1/login)
+                #TOKEN=${CURLRESPONSE:10:32}
+                #curl -H "x-molgenis-token:${TOKEN}" -X POST -F"file=@${LOGDIR}/${_project}/jobsPerProject.tsv" -FentityName='status_jobs' -Faction=add -Fnotify=false https://${MOLGENISSERVER}/plugin/importwizard/importFile
+                #echo "curl -H "x-molgenis-token:${TOKEN}" -X POST -F"file=@${LOGDIR}/${_project}/jobsPerProject.tsv" -FentityName='status_jobs' -Faction=add -Fnotify=false https://${MOLGENISSERVER}/plugin/importwizard/importFile"
+##
+#######################################
+
+                sleep 10
+
+		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' \
+			"starting to submit jobs"
+		sh submit.sh
+
+		touch "${LOGDIR}"/"${_project}"/"${_project}".pipeline.started
+		echo "${_project} started" >> ${_logger}
+		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' \
+			"${_project} started"
+		printf "Pipeline: ${pipeline}\nStarttime:`date +%d/%m/%Y` `date +%H:%M`\nProject: $PROJECT\nStarted by: $WHOAMI\n \
+		Host: \${HOSTN}\n\nProgress can be followed via the command squeue -u $WHOAMI on $HOSTN.\nYou will receive an email when the pipeline is finished!\n\nCheers from the GCC :)" \
+		| mail -s "NGS_DNA pipeline is started for project $PROJECT on `date +%d/%m/%Y` `date +%H:%M`" ${ONTVANGER}
+
+		sleep 40
+		rm -f "${LOGDIR}"/"${_project}"/"${_run}".pipeline.locked
+	fi
+}
+
+#
+# Get commandline arguments.
+#
+log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Parsing commandline arguments..."
+declare group=''
+declare email='false'
+declare dryrun=''
+while getopts "g:l:hen" opt; do
+	case $opt in
+		h)
+			showHelp
+			;;
+		g)
+			group="${OPTARG}"
+			;;
+		e)
+			email='true'
+			;;
+		n)
+			dryrun='-n'
+			;;
+		l)
+			l4b_log_level=${OPTARG^^}
+			l4b_log_level_prio=${l4b_log_levels[${l4b_log_level}]}
+			;;
+		\?)
+			log4Bash "${LINENO}" "${FUNCNAME:-main}" '1' "Invalid option -${OPTARG}. Try $(basename $0) -h for help."
+			;;
+		:)
+			log4Bash "${LINENO}" "${FUNCNAME:-main}" '1' "Option -${OPTARG} requires an argument. Try $(basename $0) -h for help."
+			;;
+	esac
+done
+
+#
+# Check commandline options.
+#
+if [[ -z "${group:-}" ]]; then
+	log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' 'Must specify a group with -g.'
+fi
+if [[ -n "${dryrun:-}" ]]; then
+	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' 'Enabled dryrun option for rsync.'
+fi
+
+
```

Comment:
I'm missing some bits here:
* sourcing of config files.
* thereShallOnlyBeOne function using flock to make sure only one instance is running (per group).
* check to make sure this script is executed using the right user account: ${group-ateambot}